Directory.Build.props

```diff
@@ -15,6 +15,7 @@
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)build\Key.snk</AssemblyOriginatorKeyFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <AssemblySigningCertName>Microsoft</AssemblySigningCertName>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Overloading between `Action` and `Func<Task>` **with** method group to delegate conversion will require C# 7.3. In practice that shouldn't be a problem since we're requiring everyone to have a version of the tools that support it. Thoughts?